lib/views/bottom-panel.js

```diff
@@ -3,49 +3,59 @@
 let Message = require('./message')
 
 class BottomPanel extends HTMLElement{
-  prepare(){
+
+  prepare() {
     // priority because of https://github.com/atom-community/linter/issues/668
     this.panel = atom.workspace.addBottomPanel({item: this, visible: false, priority: 500})
     this.panelVisibility = true
     return this
   }
-  destroy(){
+
+  destroy() {
     this.panel.destroy()
   }
-  get panelVisibility(){
+
+  get panelVisibility() {
     return this._panelVisibility
   }
-  set panelVisibility(value){
+
+  set panelVisibility(value) {
     this._panelVisibility = value
-    if(value) this.panel.show()
+    if (value) this.panel.show()
     else this.panel.hide()
   }
-  get visibility(){
+
+  get visibility() {
     return this._visibility
   }
-  set visibility(value){
+
+  set visibility(value) {
     this._visibility = value
-    if(value){
+    if (value) {
       this.removeAttribute('hidden')
     } else {
       this.setAttribute('hidden', true)
     }
   }
-  updateMessages(messages, isProject){
+
+  updateMessages(messages, isProject) {
     this.clear()
-    if(!messages.length){
-      return this.visibility = false
+    if (messages.length) {
```

Comment:
It was `!messages.length` before and you made it `messages.length`. Bad typo. cost me hours!